build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh

```diff
@@ -72,7 +70,26 @@ ninja
 
 export CTEST_PARALLEL_LEVEL=${CTEST_PARALLEL_LEVEL:-$(nproc)}
 
-echo "Testing with CTest"
-ctest --timeout 900 --output-on-failure \
+tests_passed=true
+
+echo "***** Testing with CTest *****"
+if ! ctest --timeout 900 --output-on-failure \
```

Comment:
What's still being tested with ctest here?

Reply:
A lot of stuff. All the tests for the non-integrations part of the project, I believe.

Reply:
This build is specifically for the integrations part of the project though (which needs Bazel). Everything else is in the core build.

Reply:
We should look at the logs. I think I'm remembering seeing that it is indeed still running some things.